src/components/implementation/tests/unit_defcompinfo/unit_defcompinfo.c

```diff
@@ -199,7 +201,7 @@ cos_init(void)
 
 		/* TEST BLOCKING */
 		/* TODO: Challenge - how does a component know at runtime if can call cos_rcv or not? - It does not at runtime. */
-		cos_rcv(BOOT_CAPTBL_SELF_INITRCV_BASE, 0, NULL);
+		cos_rcv(BOOT_CAPTBL_SELF_INITRCV_BASE, 0, 0, NULL);
```

Comment:
I've taken out the `timeout` parameter from `cos_rcv`, do you still think this is something we should look into even without that?